content/en/docs/community/_index.md

```diff
@@ -14,5 +14,5 @@ menu:
 
 MIMIC is provided through the work of researchers at the MIT Laboratory for Computational Physiology and [our collaborators](/docs/about/acknowledgments/). We have limited resources and cannot provide individual support to researchers worldwide. We therefore encourage MIMIC users to work together as a community.
 The [MIMIC Code Repository](https://github.com/MIT-LCP/mimic-code/) contains collaboratively developed code for a large number of useful concepts in MIMIC.
-The issues page provides a forum for public discussion which reaches a larger audience, enables wider discourse, and full transparency around discussions.
+The issues page provides a forum for [public discussion](https://github.com/MIT-LCP/mimic-code/discussions) which reaches a larger audience, enables wider discourse, and full transparency around discussions.
```

Comment:
Thanks @danamouk , adding this link is a good idea!  Perhaps we should start the sentence with "The discussions page provides" though.

Reply:
I have changed the text to include discussions page instead of issues page.